Earthfile

```diff
@@ -266,6 +267,13 @@ perl:
   RUN apk add --no-cache perl
   DO +BENCH --name="perl" --lang="Perl" --version="perl -v" --cmd="perl leibniz.pl"
 
+pony:
+  FROM ponylang/ponyc:alpine
+  DO +PREPARE_ALPINE
+  DO +ADD_FILES --src="pony/*.pony"
```

Comment:
Does it need to be in a separate folder in `src/`? I would prefer them all to be just in the root folder if possible. An exception is C# because it needs the project metadata file.

Reply:
Because of ponyc compiles a folder of the source code. So I just place this as a folder for easily compiling. But for now I'll take this source to `src` for the source has only one file.